lenny/app.py

```diff
@@ -12,9 +14,14 @@
     version=VERSION,
 )
 
+create_s3()
+create_db()
```

Comment:
```suggestion
```

These can actually be removed since now they are automatically initialized in the constructor of `LennyDB` and `LennyS3`